components/buttons/MismatchButton.tsx

```diff
@@ -228,8 +235,12 @@ const MismatchNotice: React.FC<{
         isDisabled={!actuallyCanAttemptSwitch}
         colorScheme="orange"
         textTransform="capitalize"
+        noOfLines={1}
```

Comment:
won't this alone already do it?

Reply:
it gets cut out (doesn't respect paddings) so I went through the convoluted solution but still added that for mobile or edge cases)

basically it's the "this ensures it never happens no matter what"